### DISCUSSION TITLE
[mastodon.social] Password reset mail / gmail problems?

### DISCUSSION THREAD

**Discussion by iamwebrocker** [mastodon/mastodon#19553] [Q&A] [answered]
Hi,

I created my account @webrocker@mastodon.social in 2018, and I'm pretty sure that I used my gmail address back then, quite possibly with an "+mastodon" suffix.
Now when I try to log in, the mail/pw combination is wrong and the pw reset mail never comes through. I tried every email address I can think of - no mail. 
Now I read somewhere that mastodon.social blocks @gmail addresses for account _creation_ because of the high amount of spam/fake accounts they are getting from that mail domain.

Is there a possibility that this also affects _existing_ accounts?

What is the best route to get the pw resetted?

thanks

[Edit]
**Solved. The one address I completely forgot to try, was the right one. D'oh. Sorry for the bother.**

**Comment by Gargron** [comment, marked as answer]
> Now I read somewhere that mastodon.social blocks @gmail addresses for account creation because of the high amount of spam/fake accounts they are getting from that mail domain.

No we do not do that, and never did! Probably confusing us with another Mastodon server.

You can reach out to staff@mastodon.social and I'll try to help.

**Reply by iamwebrocker** [reply]
Thanks for the quick answer and support <3

**Reply by iamwebrocker** [reply]
Thank you -- I just received the PW reset mail. Completely case of problem exists between keyboard and monitor.
